alf/utils/common.py

```diff
@@ -100,12 +101,15 @@ def get_target_updater(models, target_models, tau=1.0, period=1):
         A callable that performs a soft update of the target model parameters.
     """
     models = as_list(models)
-    target_models = as_list(models)
+    target_models = as_list(target_models)
 
     def update():
+        update_ops = []
         for model, target_model in zip(models, target_models):
-            tfa_common.soft_variables_update(model.variables,
-                                             target_model.variables, tau)
+            update_op = tfa_common.soft_variables_update(
+                model.variables, target_model.variables, tau)
+            update_ops.append(update_op)
+        return tf.group(*update_ops)
```

Comment:
does tf.group have any effect in tf 2.0?

Reply:
here required  return a tf operator when tf.functions is used